zone/raids.cpp

```diff
@@ -1249,13 +1354,14 @@ void Raid::SendGroupUpdate(Client *to)
 
 void Raid::GroupUpdate(uint32 gid, bool initial)
 {
-	if(gid > 11) //ungrouped member doesn't need grouping.
+	if(gid > 11) {//ungrouped member doesn't need grouping.
 		return;
-	for(int x = 0; x < MAX_RAID_MEMBERS; x++)
+	}
+	for (int x = 0; x < MAX_RAID_MEMBERS; x++)
```

Comment:
You can range through the array versus doing the integer iteration

eg

```
for (auto & m : members) {
```

Accessing

```cpp
m.member->DoThing()
```